doc/requirement-checker.md

```diff
@@ -135,6 +135,16 @@ $ BOX_REQUIREMENT_CHECKER=0 php box.phar
 ```
 
 
+## Logging to stdout
+
+Since version x.x.x, box logs the requirements checker output to `stderr` per default (see: [#678](https://github.com/box-project/box/issues/678)).
```

Comment:
```suggestion
Since version 3.17.0, box logs the requirements checker output to `stderr` per default (see: [#678](https://github.com/box-project/box/issues/678)).
```